analyzers/SonarAnalyzer.sln

```diff
@@ -9,6 +9,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common Solution Items", "Co
 		..\azure-pipelines.yml = ..\azure-pipelines.yml
 		NuGet.Config = NuGet.Config
 		src\Signing.cs = src\Signing.cs
+		packaging\SonarAnalyzer.CSharp.nuspec = packaging\SonarAnalyzer.CSharp.nuspec
+		packaging\SonarAnalyzer.VisualBasic.nuspec = packaging\SonarAnalyzer.VisualBasic.nuspec
```

Comment:
..and CFG too please :)

Reply:
It is already since it's part of the `SonarAnalyzer.CFG` project: `src\SonarAnalyzer.CFG\SonarAnalyzer.CFG.cs.nuspec`. These too are in a separate project.